__tests__/__snapshots__/compatibility.ts.snap

```diff
@@ -224,18 +224,6 @@ Object {
       "title": "P1",
       "url": "http://p1.com",
     },
-    Object {
```

Comment:
Not sure why this post was even part of the feeds as it was part of a private source. This is true everywhere